src/main/java/org/terasology/climateConditions/FrostbiteSystem.java

```diff
@@ -57,18 +54,18 @@
     private static final int healthDecreaseAmount = 15;
     private Random random = new FastRandom();
 
-    @ReceiveEvent(components = {HypothermiaComponent.class})
-    public void onHypothermia(OnAddedComponent event, EntityRef player) {
+
```

Comment:
guess that can be removed?

Reply:
That is removed 😅. If you're asking .. whether it is alright to remove it, the yes, applyFrostbite() will be called at the time  of change in Hypothermia Levels